var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -26,7 +27,7 @@ class Timemory(CMakePackage):
     variant('caliper', default=True, description='Enable Caliper support')
     variant('gperftools', default=True, description='Enable gperftools support')
 
-    patch('0001-handle-CUPTI-API-11-fix-disable-build-of-mpip-if-req.patch')
+    patch('0001-handle-CUPTI-API-11-fix-disable-build-of-mpip-if-req.patch', when='@3.0.0a')
```

Comment:
As there is proper release available, I would remove our custom version `3.0.0a` and remove patch file.

Reply:
done